crates/pallet-evm-balances/src/imbalances.rs

```diff
@@ -60,7 +60,10 @@ impl<T: Config<I>, I: 'static> Imbalance<T::Balance> for PositiveImbalance<T, I>
 
     fn split(self, amount: T::Balance) -> (Self, Self) {
         let first = self.0.min(amount);
-        let second = self.0 - first;
+        let second = self
+            .0
+            .checked_sub(&first)
+            .expect("valid operation due to the check before; qed.");
```

Comment:
The type bounds for `T::Balance` specify it as unsigned, so this code doesn't panic and would be equivalent (in the meaning as well). However, the current version is better in that it's as strict as possible.

```suggestion
        let second = self.0.saturating_sub(amount);
```

Reply:
Usually we prefer more strict way as possible. A lot of examples can be found at `pallet-balanced-curreny-swap-bridges-initializer`.